http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeV2ServiceInterp.scala

```diff
@@ -308,6 +308,16 @@ class RuntimeV2ServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
         .raiseError[Unit](RuntimeNotFoundException(runtime.cloudContext, runtimeName, "permission denied"))
         .whenA(!hasPermission)
 
+      // Query WSM for Landing Zone resources
```

Comment:
`DeleteAzureRuntimeMessage` now needs `LandingZoneResources`, since it needs the relay namespace in order to delete the hybrid connection. This is a similar pattern to `CreateAzureRuntimeMessage`.